PythonAPI/pycocotools/cocoeval.py

```diff
@@ -503,8 +503,8 @@ def setDetParams(self):
         self.imgIds = []
         self.catIds = []
         # np.arange causes trouble.  the data point on arange is slightly larger than the true value
-        self.iouThrs = np.linspace(.5, 0.95, np.round((0.95 - .5) / .05) + 1, endpoint=True)
-        self.recThrs = np.linspace(.0, 1.00, np.round((1.00 - .0) / .01) + 1, endpoint=True)
+        self.iouThrs = np.linspace(.5, 0.95, int(np.round((0.95 - .5) / .05)) + 1, endpoint=True)
```

Comment:
Tempting to rewrite this as
```suggestion
        self.iouThrs = np.arange(1, 19+1) / 20
```

Reply:
Did you mean `np.arange(10, 19+1) / 20`?

Reply:
Other than that, I agree with your suggestion to use some form of `arange` instead of the floating point shenanigans in `np.round((0.95 - .5) / .05)`.

Reply:
Fixed the typo above, also for the hundredths case - good catch.

Reply:
Best use floats in the division for Python 2.7 compatibility.